fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb

```diff
@@ -14,11 +14,17 @@ def self.run(params)
         end
         UI.user_error!("Please provide an api_token using api_token:") unless params[:api_token]
 
-        command = []
-        command << File.expand_path(params[:binary_path])
-        command << "-a #{params[:api_token]}"
-        command << "-p #{params[:platform]}"
-        command << File.expand_path(params[:dsym_path])
+        dsym_paths = []
+        dsym_paths << params[:dsym_path]
```

Comment:
does that mean that the dsym_path param could become optional ? could it be skipped if nil ?